kover-gradle-plugin/build.gradle.kts

```diff
@@ -40,20 +38,18 @@ kotlin.target.compilations.run {
 dependencies {
     // exclude transitive dependency on stdlib, the Gradle version should be used
     compileOnly(kotlin("stdlib"))
+    compileOnly(libs.gradlePlugin.kotlin)
+    compileOnly(libs.intellij.reporter)
 
-    compileOnly("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
-
-    compileOnly("org.jetbrains.intellij.deps:intellij-coverage-reporter:1.0.721")
-
-    testImplementation(kotlin("test"))
-
-    "functionalTestImplementation"("org.junit.jupiter:junit-jupiter:5.9.0")
-    "functionalTestImplementation"("org.junit.jupiter:junit-jupiter-params:5.9.0")
+    "functionalTestImplementation"(kotlin("test"))
```

Comment:
is it possible to use a separate variable for configuration instead of using `Stirng.invoke()`? I think it would be more readable